src/Exception/SlackErrorResponse.php

```diff
@@ -15,17 +15,26 @@
 
 class SlackErrorResponse extends \Exception
 {
-    private $errorCode;
+    private string $errorCode;
 
-    public function __construct(string $errorCode, \Throwable $previous = null)
+    private ?array $responseMetadata;
```

Comment:
Sadly, typed properties can not be added yet as we still support PHP 7.2 (and they are available from PHP 7.4 only). Could you remove them?

Reply:
I believe the `__construct` ones work, they were already in there when I coded this, just the `private` variable definitions at the top. I've removed those now!